.github/workflows/docker-image.yml

```diff
@@ -33,44 +42,44 @@ jobs:
     strategy:
       matrix:
         include:
-        - image: ubuntu2004-zstd
+          - image: ubuntu2004-zstd
```

Comment:
It'd be good to add all the images we plan to build to this matrix too so they will be tested after they're pushed to docker-hub